md/taxonomy.md

```diff
@@ -115,6 +115,7 @@
     * [Configure a test organization](configure-a-test-organization.md)
     * [Run a process from Bonita Studio for testing](run-a-process-from-bonita-bpm-studio-for-testing.md)
     * [Log files](log-files.md)
+  * [Project deployment in Studio](project_deploy_in_studio.md)
```

Comment:
```suggestion
  * [Project deployment in Bonita Development Suite](project_deploy_in_dev_suite.md)
```

Reply:
I think we have talked about the "Development Suite" naming once. 
Can't remember why we did not decide to go for it eventually.
In case you don't go for it, replace "Project deployment in Studio" by "Project deployment from the studio" and also the name of the page.